pages/people/[id]/warning-notes/[warningNoteId]/view/[[...stepId]].tsx

```diff
@@ -0,0 +1,31 @@
+import { useRouter } from 'next/router';
+
+import PersonView from 'components/PersonView/PersonView';
+import BackButton from 'components/Layout/BackButton/BackButton';
+import WarningNoteRecap from 'components/WarningNote/WarningNoteRecap/WarningNoteRecap';
+
+const ViewWarningNote = (): React.ReactElement => {
+  const { query } = useRouter();
+  const personId = Number(query.id as string);
+  const warningNoteId = Number(query.warningNoteId as string);
+
+  return (
+    <>
+      <BackButton />
+      <h1 className="govuk-fieldset__legend--l gov-weight-lighter">
+        {'Warning Note Details'}
```

Comment:
JSX is a little magic with strings and you don't need to wrap them with `{}`:

```suggestion
        Warning Note Details
```

Reply:
Done in 1eeb478